src/main/views/subscription-search-case-results.njk

```diff
@@ -0,0 +1,50 @@
+{% extends "template.njk" %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{% from "macros/common-components.njk" import submitButton %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "macros/common-components.njk" import goBack %}
+
+{% block pageTitle %}
```

Comment:
add {{super()}} call to this block to render user navigation

Reply:
added.